pkg/kubelet/kubelet_pods_test.go

```diff
@@ -319,12 +319,28 @@ fe00::2	ip6-allrouters
 # Entries added by HostAliases.
 123.45.67.89	foo	bar	baz
 456.78.90.123	park	doo	boo
+`,
+		},
+		{
+			[]string{"203.0.113.1", "fd00::/6"},
```

Comment:
shouldn't this be IP addresses only, instead of subnets? 
`fd00::6/` instead of `fd00::/6`